cvat/apps/engine/models.py

```diff
@@ -167,7 +167,7 @@ class Task(models.Model):
         on_delete=models.SET_NULL, related_name="assignees")
     bug_tracker = models.CharField(max_length=2000, blank=True, default="")
     created_date = models.DateTimeField(auto_now_add=True)
-    updated_date = models.DateTimeField(auto_now_add=True)
+    updated_date = models.DateTimeField(auto_now=True)
```

Comment:
This change requires creating a migration. Could you please add a migration file?

Reply:
resolved! pls review again!